ansible/roles/package-upgrade/tasks/partials/brew.yml

```
@@ -36,10 +36,20 @@
     script: files/install-homebrew.sh
     when: os == "macos11.0" and not armbrew.stat.exists
 
+  # Ignoring deprecation errors in MacOS 10.x
+  - name: Upgrade installed packages 
+    become_user: "{{ ansible_user }}"
+    homebrew:
+      upgrade_all: yes
+    when: os == "macos10.15" || os == "macos10.14"
+    ignore_errors: true
```

Instead of creating a duplicate block, you could just add
```suggestion
    ignore_errors: os == "macos10.15" || os == "macos10.14"
```
to the original block.

---

Great suggestion @richardlau! I fix it in [0d7a28a](https://github.com/nodejs/build/pull/3130/commits/0d7a28ad2c0c483ec0fabbc47535d5a31459a212)